### PR TITLE
data: Olusoji01

### PR DESCRIPTION
**Pull request by Olusoji01**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/Olusoji01.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Ajayi Sunday",
+  "bio": "Open Source Javascript | Frontend Developer
```
Comment (sital002) [inline data/Olusoji01.json:3]:
```suggestion
  "bio": "Open Source Javascript | Frontend Developer",
```

File: data/Olusoji01.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    {
+      "name": "Follow me on twitter",
+      "url": "https://twitter.com/ajayi_sajayi?t=omaSSRmpA38QRTSV-fzfaw&s=09",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "let's connect on linkedin",
+      "url": "https://www.linkedin.com/in/sunday-ajayi-a55220114?utm_source=share&utm_campaign=share_via&utm_content=profile&utm_medium=android_app",
+      "icon": "FaLinkedin"
+    },
```
Comment (sital002) [inline data/Olusoji01.json:14]:
```suggestion
    }
```

**Review by sital002 — CHANGES_REQUESTED** [review]
Please make the necessary changes to get your profile merged

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**